libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/guidance/signboard/api/MapboxExternalFileResolverTest.kt

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.navigation.ui.maps.guidance.signboard.api
+
+import android.content.Context
+import android.content.res.AssetManager
+import androidx.test.core.app.ApplicationProvider
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class MapboxExternalFileResolverTest {
+
+    private lateinit var ctx: Context
+    private lateinit var assetManager: AssetManager
+    private lateinit var externalFileResolver: MapboxExternalFileResolver
+
+    @Before
+    fun setUp() {
+        ctx = ApplicationProvider.getApplicationContext()
+        assetManager = ctx.assets
+        externalFileResolver = MapboxExternalFileResolver(assetManager)
+    }
+
+    @Test
+    fun `resolve font when font style normal font weight bold`() {
+        val mockFontFamily = "SignText"
+        val mockFontStyle = "Normal"
+        val mockFontWeight = 700
+
+        val typeface =
+            externalFileResolver.resolveFont(mockFontFamily, mockFontWeight, mockFontStyle)
+
+        assertNotNull(typeface)
```

Comment:
Is `notNull` enough of an assertion? Can we expand it a little bit?

Reply:
There isn't much I can retrieve from typeface. Cannot think of any other assertion.

Reply:
What about the `isBold`/`isItalic` methods?

Reply:
nope they always return false